updatecli/scripts/dateadd.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/bin/bash
```

Otherwise `shellcheck` complains on the Line 3:

```
set -eux -o pipefail
            ^-- [SC3040](https://www.shellcheck.net/wiki/SC3040) (warning): In POSIX sh, set option pipefail is undefined.
```